src/VsExtension/VsEvents/OnBuildPackageRestorer.cs

```diff
@@ -715,8 +715,9 @@ private void LogToVS(VerbosityLevel verbosityLevel, string message)
                 // Switch to main thread to update the progress dialog, output window or error list window
                 await ThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
 
-                // Only show messages with VerbosityLevel.Normal, VerbosityLevel.Minimal or VerbosityLevel.Quiet
-                if (verbosityLevel <= VerbosityLevel.Normal)
+                // Only show messages with VerbosityLevel.Normal. That is, info messages only.
+                // Do not show errors, warnings, verbose or debug messages on the progress dialog
+                if (verbosityLevel == VerbosityLevel.Normal)
```

Comment:
where do we respect the verbosity level from visual studio, to make verbose stuff show up?